packages/builder/src/builder.js

```diff
@@ -515,10 +515,8 @@ export default class Builder {
             templateFunction({
               ...templateVars,
               ...templateFile,
-              options: {
-                ...templateVars.options,
-                ...templateFile.options
-              }
+              options: templateFile.options || {},
+              nuxtOptions: templateVars.options
```

Comment:
Can we directly use `this.nuxt.options` here are remove it from templateVars? Could be useful for feature developments.